build_tools/i18n/crowdin.py

```diff
@@ -326,8 +326,18 @@ def _locale_data_from_csv(file_data):
     for row in file_data:
         if len(row.keys()) == 0:
             return json
-        # First index is Identifier, Third index is the translation
-        json[row["Identifier"]] = row["Translation"]
+        # First index is Identifier
+        # Third index is the translation, but might have a key 'None'
+        json[row["Identifier"]] = row.get("Translation") or row.get(None)
+
+        # hack - sometimes it's a list?
+        if isinstance(json[row["Identifier"]], list):
+            json[row["Identifier"]] = json[row["Identifier"]][0]
+
+        # hack - sometimes it's None?
+        if not json[row["Identifier"]]:
+            json[row["Identifier"]] = ''
+
```

Comment:
@nucleogenesis I'm not sure why, but I had to add the hacks above in order to get things working